### DISCUSSION TITLE
🖼️ Branding: Logos etc.

### DISCUSSION THREAD

**Discussion by pschmitt** [zdharma-continuum/zinit#17] [Ideas]
I know it's early but nevertheless this feels important to me.

We need a visual identity.

- the zdharma-continuum org needs a profile pic
- zinit deserves a proper logo

Is someone able to help in this regard? My gimp skills suck :)

And does someone have the original artwork at hand maybe?

**Comment by pschmitt** [comment]
256x256
![logo](https://user-images.githubusercontent.com/37886/140884110-937aebfb-dcd2-4abf-9921-fc6de5198234.png)

128x128 with background:
![zinit-square-128](https://user-images.githubusercontent.com/37886/140612158-ff5554b1-5231-49df-836c-0660a71a308b.png)

SVG versions:

<img src="https://raw.githubusercontent.com/gist/pschmitt/1822fc0a048615be09a0849e48b041e8/raw/ff6bac10998900328e331944d306272e976a291b/zinit-logo.svg"/>

<img src="https://raw.githubusercontent.com/gist/pschmitt/1822fc0a048615be09a0849e48b041e8/raw/6d1780a16658345d07639607c4976fb96c02dcac/zinit-logo-2.svg"/>

**Comment by alichtman** [comment]
Can you upload higher quality JPGs?

**Reply by pschmitt** [reply]
I would love to, I don't have anything more high-res than what's currently in the repo. That's why I asked whether anyone has the original artwork.

Best we could do is try to recreate the logo using the crappy svg from above... :-/

**Reply by alichtman** [reply]
A blurry logo looks worse than no logo, imo. Let's pin this until someone has time to make a new one.

**Reply by pschmitt** [reply]
The 256x256 was in the repo, [in the images branch](https://github.com/zdharma-continuum/zinit/tree/images)! I updated my post from above and set the profile picture.
